### ISSUE TITLE
no_implicit_parens strict:false failing due to erroneous newLine

### ISSUE THREAD

**Issue by Sinetheta** [clutchski/coffeelint#288]
I figured it was time to use

```
"no_implicit_parens": {
    "name": "no_implicit_parens",
    "level": "warn",
    "strict": false
},
```

in production. But I see that although it works fine in tests, it behaves differently when run on real code. A quick trace shows that `newLine: true` is showing up on tokens which I wouldn't expect. For instance:

```
foo.bar 5
```

should raise a warning, but when I log out the tokens it sees, I get

```
[ 'NUMBER',
  '5',
  { first_line: 34,
    first_column: 10,
    last_line: 34,
    last_column: 10 },
  newLine: true ]
```

which leads [the rule](https://github.com/clutchski/coffeelint/blob/master/src/rules/no_implicit_parens.coffee#L38) to think we're looking at a multiline function and that's it's okay to let it pass. Any idea what could be causing this?
